pan3d/utils.py

```diff
@@ -65,24 +65,24 @@ async def coroutine():
     ],
     "dataset_path": None,
     "more_info_link": None,
-    "array_active": None,
+    "active_array": None,
     "data_vars": [],
     "data_attrs": [],
     "coordinates": [],
     "expanded_coordinates": [],
     "x_array": None,
-    "x_scale": 1,
     "y_array": None,
-    "y_scale": 1,
     "z_array": None,
-    "z_scale": 1,
     "t_array": None,
     "t_index": 0,
     "t_max": 0,
-    "view_edge_visibility": True,
     "error_message": None,
     "da_size": None,
     "mesh_timeout": 30,
+    "dialog_shown": None,
+    "dialog_message": None,
+    "selected_config_file": None,
+    "state_export": None,
 }
```

Comment:
All those state variable names should be group and easy to spot which part they belong to (ui_*, da_*, ...)

Reply:
Variables renamed in [6f497a9](https://github.com/Kitware/pan3d/pull/33/commits/6f497a9be4f3e7e817c6b87aa61a51dcd2ea71bf)

Reply:
love it, much easier to grasp what those vars are for.